engine/src/execution_graph/logic_controllers/LogicalProject.cpp

```diff
@@ -481,6 +481,51 @@ std::unique_ptr<cudf::column> evaluate_string_functions(const cudf::table_view &
         }
         break;
     }
+    case operator_type::BLZ_CAST_TIMESTAMP_SECONDS:
```

Comment:
This code is very repetitive. Could you please refactor it so that its more like:
```
case operator_type::BLZ_CAST_TIMESTAMP_SECONDS:
case operator_type::BLZ_CAST_TIMESTAMP_MILLISECONDS:
case operator_type::BLZ_CAST_TIMESTAMP_MICROSECONDS:
case operator_type::BLZ_CAST_TIMESTAMP:
//do common logic
//have just one small case statement to pick the right cudf::data_type
```